### PR TITLE
rich workspaces

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
- show button to create folder info
- bug fixes

Signed-off-by: tobiasKaminsky <tobias@kaminsky.me>

### CODE REVIEW

**Comment by tobiasKaminsky** [comment]
@jancborchardt @marinofaggiana

**Comment by juliushaertl** [comment]
Tested and works, also fixes the file creation with OO

File: src/main/java/com/owncloud/android/ui/fragment/OCFileListFragment.java [hunk 1]
```diff
@@ -789,6 +790,31 @@ public void onPrepareOptionsMenu(@NonNull Menu menu) {
...
+                RemoteOperationResult result = new RichWorkspaceDirectEditingRemoteOperation(mFile.getRemotePath())
+                    .execute(accountManager.getUser().toPlatformAccount(), requireContext());
+
+                if (result.isSuccess()) {
+                    String url = (String) result.getSingleData();
+                    mContainerActivity.getFileOperationsHelper().openRichWorkspaceWithTextEditor(mFile,
+                                                                                                 url,
+                                                                                                 requireContext());
+                } else {
+                    DisplayUtils.showSnackMessage(getView(), "Error");
```
Comment (AndyScherzinger) [inline src/main/java/com/owncloud/android/ui/fragment/OCFileListFragment.java]: please use a `strings.xml` `@StringRes´  string resource that can be translated :)

File: src/main/res/layout/text_file_preview.xml [hunk 2]
```diff
@@ -16,19 +16,18 @@
   along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 -->
-<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (AndyScherzinger) [inline src/main/res/layout/text_file_preview.xml]: why did you drop the scrollview?

Comment (AndyScherzinger) [reply src/main/res/layout/text_file_preview.xml]: Ans will this work for non rich-text previews (if they still exist?)?

**Comment by tobiasKaminsky** [comment]
Lint: 
Attribute unused on older versions
Vector Image Generation
Both warning that "autoMirrored" might not work, but this is acceptable for the icon, so manually increasing lint.

**Pull request merged**